test-it/ViaTests/TestVI_GetVariantAttribute_IncompatibleClusterDataTypeReportsError.via

```diff
@@ -0,0 +1,85 @@
+// Autogenerated Vireo assembly file
```

Comment:
This VI verifies that an error with the right error code is reported on VIs when incompatible data types are wired to the VariantToData node.
The following cases are tested:

1. Variant of Int32 vs String
2. Variant of Int32 vs Cluster of Int32
3. Variant of cluster of Int32 vs Int32
4. Variant of cluster of scalars vs cluster with same scalar data types but in different order
5. Variant of string vs Path
6. Variant of 3D array of Int32 vs 3D array of strings
7. Variant constant vs int32
8. Variant of 2D array of Int32 vs 1D array of Int32
9. Existing errors on the error wire are retained in case of no new errors
10. Unwired error input does not report error
11. Unwired data output does not report error